Tests/Unit/ZoomApiServiceTest.php

```diff
@@ -0,0 +1,265 @@
+
+
+        $result = $service->getRecordings('2023-01-01', '2023-01-02');
+
+        $this->assertEquals(['cached123'], $result);
+    }
+
+    /** @test */
+    public function getRecordingsWithStringDatesEmptyCacheReturnsFetchedData()
+    {
+        $handlerStack = HandlerStack::create(
+            new MockHandler([
+                new Response(200, [], json_encode(['meetings' => ['meeting1'], 'next_page_token' => '1234567890'])),
+                new Response(200, [], json_encode(['meetings' => ['meeting2'], 'next_page_token' => ''])),
+            ])
+        );
+        $client = new Client(['handler' => $handlerStack]);
+
+        $service = $this->getService($client);
+        $service->initializeObject();
+        $this->cacheMock->method('get')->willReturn(false);
+
+
+        $result = $service->getRecordings('2023-01-01', '2023-01-02');
+
+
+        $this->assertEquals(['meeting1', 'meeting2'], $result);
+    }
+
+    /** @test */
+    public function getRecordingsWithObjectDatesEmptyCacheReturnsFetchedData()
+    {
+        $handlerStack = HandlerStack::create(
+            new MockHandler([
+                new Response(200, [], json_encode(['meetings' => ['meeting1'], 'next_page_token' => '1234567890'])),
+                new Response(200, [], json_encode(['meetings' => ['meeting2'], 'next_page_token' => ''])),
+            ])
+        );
+        $client = new Client(['handler' => $handlerStack]);
+
+        $service = $this->getService($client);
+        $service->initializeObject();
+        $this->cacheMock->method('get')->willReturn(false);
+
+
+        $result = $service->getRecordings(new DateTime('2023-01-01'), new DateTime('2023-01-02'));
+
+
+        $this->assertEquals(['meeting1', 'meeting2'], $result);
+    }
+
+    /** @test */
+    public function getRecordingsWithObjectDatesAndCacheExceptionReturnsFetchedData()
+    {
+        $handlerStack = HandlerStack::create(
+            new MockHandler([
+                new Response(200, [], json_encode(['meetings' => ['meeting1'], 'next_page_token' => ''])),
+                new Response(200, [], json_encode(['meetings' => ['meeting2'], 'next_page_token' => ''])),
+                new Response(200, [], json_encode(['meetings' => ['meeting3'], 'next_page_token' => '1234567890'])),
+                new Response(200, [], json_encode(['meetings' => ['meeting4'], 'next_page_token' => ''])),
+            ])
+        );
+        $client = new Client(['handler' => $handlerStack]);
+
+        $service = $this->getService($client);
+        $service->initializeObject();
+        $this->cacheMock->method('get')->willReturn(false);
+        $this->cacheMock->method('set')->willThrowException(new Exception());
+        $this->systemLoggerMock->shouldReceive('error')->once();
+
+        $result = $service->getRecordings(new DateTime('2023-01-01'), new DateTime('2023-03-01'));
+
+
+        $this->assertEquals(['meeting1', 'meeting2', 'meeting3', 'meeting4'], $result);
+    }
+
```

Comment:
There's a `/** @test */` missing here. 
In PHPStorm I've added a code completion to help with that - so you just have to type test and tab to insert the completion:
![image](https://github.com/code-q-web-factory/CodeQ.ZoomApi/assets/2337648/b673eafc-5f2e-4854-88be-3b6995f9373b)

Reply:
/** @test */ still missing here, so this test is never actually run..

```suggestion

/** @test */
```